examples/rmg/commented/input.py

```diff
@@ -150,6 +150,16 @@
 )
 
 options(
+    	#provides a name for the seed mechanism produced at the end of an rmg run default is 'Seed'
+    name='RMGButane',	
+	#if True every iteration it saves the current model as a libraries/seeds
+	#(and deletes the old one)
+	#Unlike HTML this is inexpensive time-wise
+	#note this one will be generated at the end of a completed run and some incomplete runs even
+	#if False
+    generateSeeds=True,
```

Comment:
I spent quite a while debating that.  My concern is that the user will think the seed is only generated at the end of the run if I name it generateSeed.  I think I'll name it instead generateSeedEachIteration, which is annoying, but clearer.